src/js/jsx/sections/transform/TransformPanel.jsx

```diff
@@ -66,15 +98,67 @@ define(function (require, exports, module) {
-                                    <SVGIcon CSSID="reference-cm" />
-                                </div>
+                            <div title="Reference point">
+                                <svg id="reference-none" preserveAspectRatio="xMidYMid" width="100%"
+                                    height="100%" viewBox="0 0 24 24">
+                                <path d="M21.000,16.000 L21.000,21.000 L16.000,21.000 L16.000,19.000
+                                    L14.000,19.000 L14.000,21.000 L9.000,21.000 L9.000,19.000
+                                    L7.000,19.000 L7.000,21.000 L2.000,21.000 L2.000,16.000 L4.000,16.000
+                                    L4.000,14.000 L2.000,14.000 L2.000,9.000 L4.000,9.000 L4.000,7.000
+                                    L2.000,7.000 L2.000,2.000 L7.000,2.000 L7.000,4.000 L9.000,4.000
+                                    L9.000,2.000 L14.000,2.000 L14.000,4.000 L16.000,4.000 L16.000,2.000
+                                    L21.000,2.000 L21.000,7.000 L19.000,7.000 L19.000,9.000 L21.000,9.000
+                                    L21.000,14.000 L19.000,14.000 L19.000,16.000 L21.000,16.000 ZM10.000,20.000
+                                    L13.000,20.000 L13.000,17.000 L10.000,17.000 L10.000,20.000
+                                    ZM3.000,17.000 L3.000,20.000 L6.000,20.000 L6.000,17.000 L3.000,17.000
+                                    ZM3.000,10.000 L3.000,13.000 L6.000,13.000 L6.000,10.000 L3.000,10.000
+                                    ZM6.000,3.000 L3.000,3.000 L3.000,6.000 L6.000,6.000 L6.000,3.000
+                                    ZM13.000,3.000 L10.000,3.000 L10.000,6.000 L13.000,6.000 L13.000,3.000
+                                    ZM16.000,5.000 L14.000,5.000 L14.000,7.000 L9.000,7.000 L9.000,5.000
+                                    L7.000,5.000 L7.000,7.000 L5.000,7.000 L5.000,9.000 L7.000,9.000
+                                    L7.000,14.000 L5.000,14.000 L5.000,16.000 L7.000,16.000 L7.000,18.000
+                                    L9.000,18.000 L9.000,16.000 L14.000,16.000 L14.000,18.000 L16.000,18.000
+                                    L16.000,16.000 L18.000,16.000 L18.000,14.000 L16.000,14.000 L16.000,9.000
+                                    L18.000,9.000 L18.000,7.000 L16.000,7.000 L16.000,5.000 ZM20.000,6.000
+                                    L20.000,3.000 L17.000,3.000 L17.000,6.000 L20.000,6.000 ZM20.000,13.000
+                                    L20.000,10.000 L17.000,10.000 L17.000,13.000 L20.000,13.000 ZM17.000,17.000
+                                    L17.000,20.000 L20.000,20.000 L20.000,17.000 L17.000,17.000 ZM9.000,9.000
+                                    L14.000,9.000 L14.000,14.000 L9.000,14.000 L9.000,9.000 ZM10.000,13.000
+                                    L13.000,13.000 L13.000,10.000 L10.000,10.000 L10.000,13.000 Z" />
+                                    <rect className={getReferenceClasses("rb")} id="rb"
```

Comment:
I think it would be better to format these `<rect>` elements the way we format the rest of our JSX elements with one attribute on each line. (That would probably be going overboard for the `<path>` above, though.)